config/database.yml

```diff
@@ -15,4 +24,4 @@ test:
 
 production:
   <<: *default
-  database: crb_prod
```

Comment:
I just did a global search for `crb_prod` and it only appears here. How did rails know where to get that production database before your change?

Reply:
in production, heroku will set the `DATABASE_URL` envar. that url will include the database name, which makes this config item unnecessary.